AcadeMeData/migrations/0002_User_test_data.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('AcadeMeData', '0001_initial'),
+    ]
+
+    def generate_user_data(apps, schema_editor):
+        from AcadeMeData.models import User
+        # from django.contrib.auth.models import User as djangoUser

```

Comment:
Why are you using the model? 
Use this:
from django.contrib.auth.models import User put it in 2nd line